watertap/unit_models/gac.py

```diff
@@ -546,6 +546,14 @@ def eq_isobaric_adsorbed_contam(b, t):
             doc="Elapsed time between GAC replacement in adsorber bed in operation",
         )
 
+        self.bvt = Var(
```

Comment:
```suggestion
        self.bed_volumes_treated = Var(
```
We generally try to be more explicit about naming (even though bvt is pretty convenient).